src/examples/hotgym/HelloSPTP.hpp

```diff
@@ -16,7 +16,7 @@ class BenchmarkHotgym {
 public:	
   Real64 run(
     UInt EPOCHS = 5000,
-    bool useSPlocal=true, //can toggle which (long running) components are tested, default all
+    bool useSPlocal=false, //can toggle which (long running) components are tested, default all
```

Comment:
nit, please keep the local inh SP enabled for the test, it's one of the places ensuring its code runs compatible.

Reply:
> please keep the local inh SP enabled

Ok, I had turned it off when comparing with napi_sine.  Forgot to turn it back on.  Good catch.